src/landsat_importer/landsat_metadata.py

```diff
@@ -44,33 +44,23 @@ def scan(self, bands, band_suffixes):
         for file in files:
             for band in bands:
                 suffix = band_suffixes[band]
-                if file.startswith(stem) and file.endswith(suffix):
-                    if band in band_paths:
-                        self.logger.warning("Found multiple TIF files with same suffix %s for band %s, ignoring %s"% (suffix,band,file))
+                if file == stem + "_" + suffix:
                     band_paths[band] = os.path.join(folder, file)
         return band_paths
 
-    def is_bt(self, band):
-        return band in ["10", "11"]
-
-    def is_radiance(self, band):
-        return band in ["1", "2", "3", "4", "5", "6", "7", "8", "9"] and self.oli_format is OLIFormats.RADIANCE
-
-    def is_reflectance(self, band):
-        return band in ["1", "2", "3", "4", "5", "6", "7", "8", "9"] and self.oli_format is OLIFormats.REFLECTANCE
```

Comment:
These functions could remain in the superclass if they were written something like:
```python
def is_radiance(self, band):
    return not self.is_bt(band) and self.oli_format == OLIFormats.RADIANCE
```

* oli_formats might be renamed to `band_format` or `visnir_format` to avoid using "oli" for satellites which do not have an OLI  (if we called it "visnir_format" that would be clear and leave open the possibility that we might want to add "ir_format" in the future - allowing the thermal channels to be exported as radiance or BT)
* sub classes now only need to override is_bt / is_level1_bt
* do we need to distinguish "level1_bt" and level 2 bt?  i.e. just have (for landsat 8/9) is_bt: return band in ['B10', 'B11']